src/chrome/komodo/content/pref/pref-syntax-checking.xul

```diff
@@ -147,6 +147,18 @@
                       </groupbox>
                     </vbox>
                 </groupbox>
+                <groupbox orient="vertical">
+                    <caption label="&eslintChecking.caption;"/>
+                    <description>
+                        &eslintChecking.description;
+                    </description>
+                    <hbox align="center">
+                         <checkbox id="lint_eslint_enabled"
+                            pref="true"
+                            preftype="boolean"
+                            label="&eslintChecking.checkbox;" />
+                    </hbox>
+                </groupbox>
```

Comment:
We'd want a pref here to let people configure what eslint executable to use. Can't just assume someone is working from the parent directory of the relevant node_modules folder.